include/sys/Makefile.am

```diff
@@ -107,6 +107,7 @@ COMMON_H = \
 	$(top_srcdir)/include/sys/zfs_file.h \
 	$(top_srcdir)/include/sys/zfs_fuid.h \
 	$(top_srcdir)/include/sys/zfs_project.h \
+	$(top_srcdir)/include/sys/zfs_quotas.h \
```

Comment:
It's a little thing, but can we drop the `s` from `zfs_quotas.h` and `zfs_quotas.c`.  So we instead name these `zfs_quota.h` and `zfs_quota.c`.  This would be more consistent with the existing naming convention.  You'll want to make sure to fix the guard macros as well `_SYS_ZFS_QUOTAS_H` -> `_SYS_ZFS_QUOTA_H`.